operator/src/main/kubernetes/kubernetes.yml

```diff
@@ -231,3 +231,10 @@ spec:
     matchLabels:
       app: kas-fleetshard-operator
 ---
+apiVersion: scheduling.k8s.io/v1
```

Comment:
This does not seem to be the way the priority class is wired to be picked by OLM bundles. I think we need to change bundle scripts and add other priority classes here (probably better) vs define this priority class directly in the bundle scripts of build pipeline as well as kas-installer. For my testing created this manually. but when I tried to with kas-installer I saw it created previous one in the script not from kubernetes.yml @shawkins  @MikeEdgar what do u suggest?

Reply:
The bundle scripts can be updated to expect it in `kubernetes.yml` and use that. We need both included in the bundle correct?

Reply:
We can just expand on what is done here to apply for `PriorityClass` resources.

https://github.com/bf2fc6cc711aee1a0c2a/kas-installer/blob/f9a5fcd65c07174adf6dd0cdf6c88bf688f39ea0/operators/generate-kas-fleetshard-olm-bundle.sh#L182-L184

Reply:
yeah, I think expanding the `kubernetes.yaml` and fixing the bundle scripts is the right approach if at all we need to deal with this again.

Reply:
Yes, I called this out in the initial pr description.  Unlike the other priority class this one is used internally, so it does logically belong in the kubernetes.yml.